paddle/fluid/framework/ir/multi_batch_merge_pass.cc

```diff
@@ -71,6 +71,8 @@ VarDesc UpdateGradVarDesc(
 std::unique_ptr<Graph> BatchMergePass::ApplyImpl(
     std::unique_ptr<Graph> graph) const {
   int num_repeats = Get<const int>(kNumRepeats);
+  std::vector<OpDesc*> all_op_descs = Get<std::vector<OpDesc*>>(kAllOpDescs);
```

Comment:
why are you adding this?

Reply:
this pass modify the OpDescs. in my first version design, other new passes should update the OpDescs cache when it changed. 

deleted.